pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -274,7 +274,7 @@ func TestPriorityQueue_AddUnschedulableIfNotPresent(t *testing.T) {
 // current scheduling cycle will be put back to activeQueue if we were trying
 // to schedule them when we received move request.
 func TestPriorityQueue_AddUnschedulableIfNotPresent_Backoff(t *testing.T) {
-	q := NewPriorityQueue(nil, nil)
+	q := NewPriorityQueueWithClock(nil, clock.NewFakeClock(time.Now()), nil)
```

Comment:
Another way to fix the issue is to use a "closed" channel to disable background goroutines: (note the difference with a "nil" channel)

```go
stopped := make(chan struct{})
close(stopped)
q := NewPriorityQueue(stopped, nil)
```

Reply:
More inclined to use channel. :)

Reply:
I prefer the current PR implementation, as this is the actual queue that we run. And there should be a separate test that makes sure that elements are moved back to the active queue once the backoff is fulfilled.

Reply:
Thanks @wgliang and @alculquicondor. Could you help double check if the racing issue is reproducible in your env (without this PR)? (may need multiple runnings)

Command:

```
go test k8s.io/kubernetes/pkg/scheduler/internal/queue -run=^TestPriorityQueue_AddUnschedulableIfNotPresent_Backoff$ --race --count=10
```

Reply:
It just succeeds for me. Since it is time sensitive, it probably only fails when the test takes more than 1s, which might be possible in an overloaded environment.